# 測距センサ(SCIBOT-X、L-MESのみ)GET
---
float64 Distance0
float64 Distance1
float64 Distance2
float64 Distance3

